x-pack/plugins/ml/public/maps/util.ts

```diff
@@ -109,6 +109,34 @@ export function getInitialAnomaliesLayers(jobId: string) {
   return initialLayers;
 }
 
+export function getInitialSourceIndexFieldLayer(sourceIndicesWithGeoFields: {
+  [key: string]: { geoFields: string[]; dataViewId: string };
+}) {
+  const initialLayers: any = [];
```

Comment:
Please replace `any` with [this](https://github.com/elastic/kibana/blob/f46a6309bd10e30e5ec303b52eb1ec46b78b896d/x-pack/plugins/maps/public/locators.ts#L156) type

Reply:
Updated in 194d15241a2663884443d87ef1f6505302ba654d